src/dials/command_line/merge.py

```diff
@@ -129,6 +129,9 @@
         .type = strings
         .help = "Dataset name to be used in MTZ file output (multiple names
             allowed for MAD datasets)"
+    additional_stats = False
+       .type = bool
+       .help = "Calculate and report the R-split statistic in the log"
```

Comment:
```suggestion
       .help = "Calculate and report the R-split statistic"
```
It's also in the html report not just the log.